### DISCUSSION TITLE
优化改善网页的加载及性能 | hengshuai's blog

### DISCUSSION THREAD

**Discussion by giscus** [ihengshuai/blog#105] [Announcements]
# 优化改善网页的加载及性能 | hengshuai's blog

如何优化网页的加载速度和性能指标,提高网站的seo

https://blog.usword.cn/article/2020/optimize-web-server.html

**Comment by ihengshuai** [comment]
性能优化请查看 [性能优化章节](https://blog.usword.cn/frontend/performance/base.html)
